Changed Release order and description of releases page (#17616)
ankidroid/Anki-Android: tools/release-description.md

> [!IMPORTANT]
> GitHub does not auto-update apps

---

## For regular users

Install `arm64-v8a` below. If it fails to install, use `Parallel.A`.

> `Parallel` builds install side-by-side with the main APK, allowing you to use different settings and profiles (via the `AnkiDroid` `directory` advanced setting & a different AnkiWeb login).

Reviewer: ```suggestion
`Parallel` builds install side-by-side with the main APK, allowing you to use different settings and profiles (via the `AnkiDroid directory` advanced setting and a different AnkiWeb login).
```

---

## For testers

The builds with `full`, `play` and `amazon` are useful for testing our builds for different app stores:

- **`full`**: F-Droid & GitHub `Parallel` apks

> - **`play`**: Google Play - missing `MANAGE_EXTERNAL_STORAGE` [app data is deleted on uninstall]

Reviewer: ```suggestion
- **`play`**: Google Play - does not have `MANAGE_EXTERNAL_STORAGE` permission as it is forbidden by Google, so app data is deleted on uninstall
```

> - **`amazon`**: Amazon - missing `CAMERA`

Reviewer: ```suggestion
- **`amazon`**: Amazon - currently same as full, historically removed `CAMERA` permission
```
